airflow/api_connexion/openapi/v1.yaml

```diff
@@ -445,27 +428,22 @@ paths:
       summary: Update DAGs
       x-openapi-router-controller: airflow.api_connexion.endpoints.dag_endpoint
       operationId: patch_dags
-      tags: [DAG]
       parameters:
         - $ref: '#/components/parameters/PageLimit'
         - $ref: '#/components/parameters/PageOffset'
         - $ref: '#/components/parameters/FilterTags'
         - $ref: '#/components/parameters/UpdateMask'
-        - name: only_active
-          in: query
-          schema:
-            type: boolean
-            default: true
-          required: false
-          description: |
-            Only update active DAGs.
-        - name: dag_id_pattern
-          in: query
-          schema:
-            type: string
-          required: false
-          description: |
-            If set, only update DAGs with dag_ids matching this pattern.
+        - $ref: '#/components/parameters/OnlyActive'
+        - $ref: '#/components/parameters/DagIdPattern'
+      tags: [DAG]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DAGCollection'
```

Comment:
```suggestion
              $ref: '#/components/schemas/DAG'
```

Reply:
@ephraimbuddy I made this change, but am wondering if you can clarify why the request body contains a DAG or DAGCollection schema? It doesn't seem like there is a DAG schema being passed in the request.

Reply:
Only `is_paused` is allowed to be updated but because of how clients work, we have to give the DAG object so that we generate a correct request body in clients. For example, we don't want to just have `is_paused` object, when `is_paused` is part of DAG and we already have a DAG object. The API doc knows to generate it in documentation correctly by filtering out readonly fields.